modules/repository-url/src/test/java/org/elasticsearch/repositories/url/RepositoryURLClientYamlTestSuiteIT.java

```diff
@@ -99,8 +105,10 @@ public void registerRepositories() throws IOException {
             try {
                 InetAddress inetAddress = InetAddress.getByName(new URL(allowedUrl).getHost());
                 if (inetAddress.isAnyLocalAddress() || inetAddress.isLoopbackAddress()) {
-                    Response createUrlRepositoryResponse = client().performRequest("PUT", "_snapshot/repository-url", emptyMap(),
-                        buildRepositorySettings(URLRepository.TYPE, Settings.builder().put("url", allowedUrl).build()));
+                    Request createUrlRepositoryRequest = new Request("PUT", "/_snapshot/repository-url");
+                    createUrlRepositoryRequest.setEntity(buildRepositorySettings(URLRepository.TYPE,
+                            Settings.builder().put("url", allowedUrl).build()));
```

Comment:
I sometimes wish we had something like `Settings.singleton("url", allowedUrl)`...